### PR TITLE
document new eirslett profile

### PR DESCRIPTION
**Pull request by PadPalon**
### Contributor checklist

- [X] My contribution is **ready to be merged** as is
- [X] I verified the modifications **render properly**
- [X] I used **spell checking**
- [X] I **used the styling and structure aids available** in [Sphinx/ResT](http://www.sphinx-doc.org/en/stable/rest.html). (Links use `` `…`_``, enumerations `#.`, lists `*`, code ``` ``…`` ```/`.. code::`, warnings .. `warning::`, etc.)
- [X] I reread the text keeping in mind that the text has to be **comprehended** fully **by the target audience**

### Description
document our use of eirslett and the new eirslett maven profile

### CODE REVIEW

File: ide/maven.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+=====
+
+Eirslett
+--------
+
+.. hint::
+    You can disable the maven-frontend-plugin by disabling the eirslett plugin, either directly or by passing the
+    ``skipEirslett`` argument. Use ``-DskipEirslett`` in the command line to do this.
+
+We use `frontend-maven-plugin`_ to install npm for the new client and run gulp to compile Less / Sass and the likes.
```
Comment (MethodenMann) [inline ide/maven.rst]: it's more the other way around: to install the new client with npm :)

Comment (MethodenMann) [reply ide/maven.rst]: suggestion :
We use `frontend-maven-plugin`_ to run various npm tasks such as installing js polyfill packages or packages of the new client and run gulp to compile Less / Sass and the likes.

Comment (PadPalon) [reply ide/maven.rst]: I like it, gonna use your version

**Comment by MethodenMann** [comment]
sorry totally lost track of this one :)

**Pull request merged**